app/models/departments.js

```diff
@@ -4,12 +4,119 @@
 const db      = require("../../app/db");
 const Promise = require("bluebird");
 
+const BaseModel = require("../lib/models").BaseModel;
+
 const DepartmentSupervisors = require("./departmentSupervisors");
 
+class Departments extends BaseModel {
+
+  constructor(data) {
+    super({
+      data: data,
+      columns: [
+        "department_id",
+        "organization",
+        "name",
+        "phone_number",
+        "created_by",
+        "active",
+        "created"
+      ]
+    })
+  }
+
+  static alterCase (departmentID, active) {
+    if (typeof active == "undefined") active = true;
+
+    return new Promise((fulfill, reject) => {
+      db("departments")
+        .where("department_id", departmentID)
+        .update({active: active})
+      .then(() => {
+        fulfill();
+      }).catch(reject);
+    });
+  }
 
+  static createOne (orgID, name, phoneNumber, userID) {
+    return new Promise((fulfill, reject) => {
+      db("departments")
+        .insert({
+          organization: orgID,
+          name: name,
+          phone_number: phoneNumber,
+          created_by: userID,
+          active: true
+        })
+      .then(() => {
+        fulfill()
```

Comment:
Return an instance of the thing you create. 

Actually, we have an instance of `create` on the BaseModel that returns an instance. If you want create methods that take attributes it might be better to do it like this: 

```javascript
static createOne (orgID, name, phoneNumber, userID) {
  return new Promise((fulfill, reject) => {
    return this.create({
      organization: orgID,
      name: name,
      phone_number: phoneNumber,
      created_by: userID,
      active: true
    })
  }
}
```

if there was a naming conflict, which there likely should be as we would want all our create methods to have the same name. So if the method on BaseModel is called createOne we could call `super.createOne` instead of `this.create`